statsmodels/discrete/discrete_model.py

```diff
@@ -769,18 +769,24 @@ def predict(self, params, exog=None, exposure=None, offset=None,
         If exposure is specified, then it will be logged by the method.
         The user does not need to log it first.
         """
-        #TODO: add offset tp
-        if exog is None:
-            exog = self.exog
-
-        if exposure is None:
-            # If self.exposure exists, it will already be in logs.
-            exposure = getattr(self, 'exposure', 0)
+        # the following is copied from GLM predict (without family/link check)
+        # Use fit offset if appropriate
```

Comment:
(Not necessarily for this PR) if this is used multiple times, it may merit a helper method.  Especially since IIRC there are some TODO notes regarding how to keep track of what has and has not been `log`ged.